fedimint-client/src/lib.rs

```diff
@@ -1487,6 +1488,31 @@ impl ClientBuilder {
         )
     }
 
+    pub async fn migrate_database(&self) -> anyhow::Result<()> {
+        // Only apply the client database migrations if the database has been
+        // initialized.
+        if let Ok(client_config) = self.load_existing_config().await {
+            for (module_id, module_cfg) in client_config.modules {
+                let kind = module_cfg.kind.clone();
+                let Some(init) = self.module_inits.get(&kind) else {
+                    bail!("Detected configuration for unsupported module id: {module_id}, kind: {kind}")
```

Comment:
We're not supposed to fail on modules we don't support, but just ignore them. Unless I'm too tired to be reviewing.

Reply:
Ack, will fix

Reply:
Fixed